tests/test_account.py

```diff
@@ -0,0 +1,173 @@
+                                             'level': 'free', 'compute_type': 'CUSTOMER_MANAGED',
+                                             'aws_region': None, 'storage_type': 'CUSTOMER_MANAGED',
+                                             'CacheQuotaSizeInGB': '25', 'secret': None,
+                                             'use_previous_account_plan': None,
+                                             'compute_secret_key': None,
+                                             'compute_access_key': None, 'defloc': None}}
+                                            )
+
+    def test_storage_acc_key(self):
+        sys.argv = ['qds.py', 'account', 'create','--storage-access-key', 'dummy']
+        print_command()
+        Connection._api_call = Mock(return_value={})
+        qds.main()
+        Connection._api_call.assert_called_with("POST", "account", {'account':
+                                            {'acc_key': "dummy",
+                                             'name': None,
+                                             'level': 'free', 'compute_type': 'CUSTOMER_MANAGED',
+                                             'aws_region': None, 'storage_type': 'CUSTOMER_MANAGED',
+                                             'CacheQuotaSizeInGB': '25', 'secret': None,
+                                             'use_previous_account_plan': None,
+                                             'compute_secret_key': None,
+                                             'compute_access_key': None, 'defloc': None}}
+                                            )
+
+    def test_storage_secret_key(self):
+        sys.argv = ['qds.py', 'account', 'create','--storage-secret-key', 'dummy']
+        print_command()
+        Connection._api_call = Mock(return_value={})
+        qds.main()
+        Connection._api_call.assert_called_with("POST", "account", {'account':
+                                            {'acc_key': None,
+                                             'name': None,
+                                             'level': 'free', 'compute_type': 'CUSTOMER_MANAGED',
+                                             'aws_region': None, 'storage_type': 'CUSTOMER_MANAGED',
+                                             'CacheQuotaSizeInGB': '25', 'secret': "dummy",
+                                             'use_previous_account_plan': None,
+                                             'compute_secret_key': None,
+                                             'compute_access_key': None, 'defloc': None}}
+                                            )
+
+    def test_aws_region(self):
+        sys.argv = ['qds.py', 'account', 'create','--aws-region', 'us-east-1']
+        print_command()
+        Connection._api_call = Mock(return_value={})
+        qds.main()
+        Connection._api_call.assert_called_with("POST", "account", {'account':
+                                            {'acc_key': None,
```

Comment:
Please follow PEP8 formatting/indentation style.

There are some existing files which are not following PEP8 completely. But we are trying to follow it as closely as we can in the new files which we are adding. So, please run `pep8` on both account.py and test_account.py.

Also, consider using an IDE like [pycharm](https://www.jetbrains.com/pycharm/) which suggest these improvements when you are typing.